bin/varnishd/mgt/mgt_child.c

```diff
@@ -352,9 +352,23 @@ mgt_launch_child(struct cli *cli)
 			memset(&sa, 0, sizeof sa);
 			sa.sa_sigaction = child_signal_handler;
 			sa.sa_flags = SA_SIGINFO;
-			(void)sigaction(SIGSEGV, &sa, NULL);
 			(void)sigaction(SIGBUS, &sa, NULL);
 			(void)sigaction(SIGABRT, &sa, NULL);
+
+#ifdef HAVE_SIGALTSTACK
+			stack_t ss;
+			size_t sz = SIGSTKSZ + 4096;
+			if (sz < mgt_param.wthread_stacksize)
+				sz = mgt_param.wthread_stacksize;
```

Comment:
So maybe a `thread_pool_altstack` parameter with experimental+needs_restart flags would be a better choice? It could have the same default value as `thread_pool_stack` and a minimum value of `SIGSTKSZ + 4096`.